.contrib/Parser/DATAS/02 - Outdoor Zones/07 Draenor/Gorgrond.lua

```diff
@@ -3171,9 +3177,12 @@ root(ROOTS.Zones, {
 						["sym"] = COMMON_TREASURE_SYM,
 					}),
 					o(231069, {	-- Strange Looking Dagger
-						["description"] = "Inside The Forgotten Caves with Pale and Spiders. Entrance at 51.40, 77.60.",
+						["description"] = "Inside The Forgotten Caves with Pale and Spiders.",
 						["questID"] = 34940,
-						["coord"] = { 53.0, 80.0, GORGROND },
+						["coord"] = {
```

Comment:
Does not parse, line 3182 must be ["coords"].

Reply:
Fixed